MyOSDevV0.7/MyOSAgentEvent.blue

```diff
@@ -0,0 +1,13 @@
+name: MyOS Agent Event
+description: MyOS-specific agent event with agent ID, timestamp, and event data
+agentId:
+  type: Text
+  description: Optional agent identifier
+id:
+  type: Integer
+  description: Optional event ID
+timestamp:
+  type: Integer
+  description: Optional timestamp for the event
+event:
+  description: Optional event node reference
```

Comment:
I think MyOS Agent, MyOS Agent Channel & MyOS Agent Event will be deprecated as soon as we move fully to timelines subscription (Not to do in this PR, just for your information)